.github/workflows/build-tauri.yml

```diff
@@ -52,14 +52,14 @@ jobs:
                   npm ci
             - name: Build (debug)
               if: github.event.inputs.environment != 'Production'
-              run: npm run tauri:build
+              run: npm run tauri:build:debug
```

Comment:
I felt that the regular "production" build should be the default.

Reply:
Can you explain what you mean by "local" in this context? By default, the build still points to localhost. My line of thinking here was "what would somebody expect the default behaviour to be if they just wanted to build their own version?" In that context, building for debugging purposes should be the exception I think.

Reply:
Sure. That's what I meant by "it's not like this is deploying directly to production". In any case, usually, developing locally, you want to get as much info for debugging errors as possible, but you're right that if the most common scenario is to run the build for a "personal production", this way makes more sense.

Reply:
Yep, "personal production" is a good way of putting it 😊. Basically I want to make the whole build/deployment process as ergonomic as possible for this kind of use case since personal users will be less inclined to understand the nuances of the build process.